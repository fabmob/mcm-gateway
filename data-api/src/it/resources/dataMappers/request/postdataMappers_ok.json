[
  {
    "internalField": "CHILD_SEAT",
    "externalField": "accessories",
    "isArray": 1,
    "format": null,
    "timezone": "",
    "containedValue": "BabySeat",
    "defaultValue": null,
    "partnerActionId": "5e71dcb3-ca60-4a2a-9896-c509ad1ca756"
  }
]