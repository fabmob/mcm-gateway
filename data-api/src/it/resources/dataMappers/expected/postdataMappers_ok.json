    {
        "dataMapperId": "c271cc6b-d759-4cb4-a5e6-c8b952d4f1ce",
        "champInterne": "CHILD_SEAT",
        "champExterne": "accessories",
        "isArray": 1,
        "format": null,
        "timezone": "",
        "containedValue": "BabySeat",
        "defaultValue": null,
        "mspActionId": "5e71dcb3-ca60-4a2a-9896-c509ad1ca756"
    }